lib/actors/actor/entity/entity.ex

```diff
@@ -698,13 +722,17 @@ defmodule Actors.Actor.Entity do
       system: %ActorSystem{name: system_name},
       actor: %Actor{id: %ActorId{name: actor_name}},
       command_name: cmd,
-      value: value
+      value: value,
+      caller: ActorId.new(name: caller_actor_name, system: system_name)
     }
 
     try do
       case lookup(system_name, actor_name) do
         {:ok, %HostActor{opts: opts}} ->
-          invoke(invocation, opts)
+          case invoke(invocation, opts) do
```

Comment:
there was a bug here when piping through multiple actors, it was returning in the invoke:

`{:ok, {:ok, {:ok, something}}}`

based on the number of nested pipes